### PR TITLE
Make puppet-wireguard compatible with Ubuntu 16.04 and later

### PR DESCRIPTION
**Pull request by ghost**
This is a preliminary change.

I have yet to set up a working connection. Please *do not merge* unless this is acknowledged as working.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
     {
       "name": "puppetlabs-stdlib",
       "version_requirement": ">= 1.0.0"
+    },
+    {
+      "name":"puppetlabs-apt",
+      "version_requirement":">= 4.0.0"
     }
   ],
   "operatingsystem_support": [
```
Comment (spacedog) [inline metadata.json:20]: Could you also add ubuntu to _operatingsystem_support_ list so we have spec tests running for Ubuntu

**Comment by spacedog** [comment]
Don't you mind to add spec tests for ubuntu ? We need to cover apt bits for install class

File: metadata.json (hunk 1 above)
Comment (ghost) [reply metadata.json:20]: Sorry, forgot this.

**Comment by ghost** [comment]
Have not yet written any of those, but I will try to. :+1:

**Comment by morremeyer** [comment]
@makmarius I set up a working interface with your current master branch, so I can confirm that it works on 18.04.1

**Comment by spacedog** [comment]
Thank you for contributing

**Pull request merged**